pandas/tests/io/test_parquet.py

```diff
@@ -448,9 +466,11 @@ class TestParquetFastParquet(Base):
     def test_basic(self, fp, df_full):
         df = df_full
 
-        # additional supported types for fastparquet
+        # additional supported types for fastparquet>=0.1.4
+        if LooseVersion(pyarrow.__version__) >= LooseVersion('0.1.4'):
```

Comment:
this is wrong, you mean fastparquet, yes? This is why the fixtures are FAR preferred

Reply:
Hi Jeff, I am fine with either however, Joris suggested for now to keep it in line with pyarrow style as per https://github.com/pandas-dev/pandas/pull/19434#discussion_r164313190. I feel it's good to keep it consistent and follow up with another PR for styling for both PyArrow and FP.

Reply:
@minggli only the 'pyarrow' -> 'fastparquet' on this line you need to change anyway (copy paste error)